package.json

```diff
@@ -1,24 +1,24 @@
 {
   "name": "business-create-ui",
-  "version": "5.3.1",
+  "version": "5.4.0",
   "private": true,
   "appName": "Create UI",
   "sbcName": "SBC Common Components",
   "scripts": {
-    "serve": "vue-cli-service serve",
-    "build": "vue-cli-service build",
-    "lint": "vue-cli-service lint",
-    "lint:nofix": "vue-cli-service lint --no-fix",
-    "test:unit": "vue-cli-service test:unit --testPathPattern --coverage",
-    "test:debug": "node --inspect-brk node_modules/.bin/vue-cli-service test:unit --testPathPattern --no-cache --watch --runInBand"
+    "dev": "vite",
+    "build": "vite build",
+    "serve": "vite preview",
```

Comment:
Maybe we should rename this "preview"? This will save some developers from entering "npm run serve" and wondering why their new code isn't running.

Or is there a standard for npm script names? (For example, I would have named dev -> serve.

Reply:
Regarding these scripts Sev, I was initially just following the migration guide on https://vueschool.io/articles/vuejs-tutorials/how-to-migrate-from-vue-cli-to-vite/: 
![image](https://github.com/bcgov/business-create-ui/assets/122301442/ab2a9831-3318-493c-a55f-9409c959705e)

However, if you'd prefer, we can use ours different. Let me know if you'd like me to change them.

Reply:
No, let's follow their guidance. We devs will have to get used to it.